### ISSUE TITLE
h2o.conf help - configuring http + https site

### ISSUE THREAD

**Issue by centminmod** [h2o/h2o#242]
I setup my first public h2o site on a KVM VPS, which has an existing Centmin Mod LEMP Nginx/PHP-FPM + MariaDB MySQL stack so ports 80 and 443 are already in use.

**Configuration**

So for `h2o 1.1.2-alpha1` I am using ports `8080`, `8081`, and `8082` all with a common web root at `/home/nginx/domains/h2ohttp2.centminmod.com/public`. This way I can test Nginx http at port 80 http://h2ohttp2.centminmod.com and Nginx https SPDY/3.1 at port 443 https://h2ohttp2.centminmod.com

**h2o.conf** file posted at https://gist.github.com/centminmod/b196f252cc37d1c97e63

**Goal**

* `8080` - want to setup non-https normal http 1.1 site = http://h2ohttp2.centminmod.com:8080/
* `8081` - h2o https HTTP/2 SSL site = https://h2ohttp2.centminmod.com:8081/
* `8082` - 2nd site - h2o https HTTP/2 SSL with different headers set `SSL Host: 8082` from 8081 site = https://h2ohttp2.centminmod.com:8082/

End result I want to achieve is

* Nginx http 80 = http://h2ohttp2.centminmod.com/flags.html
* Nginx https SPDY/3.1 443 = https://h2ohttp2.centminmod.com/flags.html
* h2o http 80 = http://h2ohttp2.centminmod.com:8080/flags.html
* h2o HTTP/2 8081 = https://h2ohttp2.centminmod.com:8081/flags.html
* h2o HTTP/2 different header 8082 = https://h2ohttp2.centminmod.com:8082/flags.html

**Problem**

Only `8081` site seems to work as I expected while:

* 8080 - when accessed at http://h2ohttp2.centminmod.com:8080/ redirects with error to https://h2ohttp2.centminmod.com:8080/
* 8082 site shows same header as port 8081 site ?

Basically, I am still confused with how h2o.conf is laid out so any help and clarification is much appreciated :)

cheers

George

**Comment by kazuho** [comment]
@centminmod Should we close this issue as a duplicate of #256?

**Comment by centminmod** [comment]
yup #256 clears this up too :)

**Comment by kazuho** [comment]
Thanks!  closing the issue.
